graphql/database/base/__tests__/RedisModel-authorization.test.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-env jest */
+import { getMockUserContext } from '../../test-utils'
+
+const user = getMockUserContext()
+
+describe('Model calls to `isQueryAuthorized`', () => {
+  afterEach(() => {
+    jest.resetModules()
+  })
+
+  it('passes correct params to `get` authorization check', async () => {
+    // Set a mock `isQueryAuthorized` method
+    const TestModelRangeKey = require('../test-utils/ExampleDynamoDBModelRangeKey')
```

Comment:
This is using the wrong model. If you want to just drop this test file altogether, I'm fine with that.